documentation/homework/week3/todo/src/Params/locals.js

```diff
@@ -0,0 +1,54 @@
+
+class Params {
+  get random_8() { // just for Id's For Now
+    let randomString = ""
+    for (let i = 0; i < 8; i++) {
+      let random = Math.floor(Math.random() * (57 - 49) + 49) // 33 126
+      randomString += String.fromCharCode(random)
+    }
+    return randomString
+  } // this function should return a random string that 8 Characters long
+
+  // localStorage section
+  get getLocals() {
+    const container = localStorage.getItem('list')
+    return JSON.parse(container)
+  }
+  saveLocal(getLocals, local_Object) {
```

Comment:
Why are you passing a class property back into a class property?

Reply:
it's just the parameter name, But you are right i should name it something else because it's still confusing. 👍

Reply:
No... if you take a look here:

https://github.com/JawharBairakdar/React/blob/007658ae5b1a44cf26dd6182eed6f2eb64fd9b2d/documentation/homework/week3/todo/src/App.js#L35

It looks like you're actually passing the class property back in as an argument. Unless I am missing something which is of course possible!